app/models/author.rb

```diff
@@ -0,0 +1,9 @@
+class Author < ApplicationRecord
+  #association
+  has_many :articles # An author can have many articles.
+
+  #validations
+  validates :name, presence: true
+  validates :email, presence: true, uniqueness: true
```

Comment:
add custom message for uniqueness failure

Reply:
changes implemented, please check latest commit.